packages/EasySchedule/src/ScheduleRunner.php

```diff
@@ -49,6 +47,7 @@ public function run(ScheduleInterface $schedule, OutputInterface $output): void
             try {
                 $event->run($schedule->getApplication());
             } finally {
+                $this->entityManager->clear();
```

Comment:
Now command executes in a separate process.

It looks this `clean` is useless.

Reply:
What do you mean by a separate process?

Reply:
Under the hood, it use `\Symfony\Component\Console\Application::run`

Reply:
Sorry my, bad. All ok, it must worked as expected.

Reply:
No, it's not a separate process. Console application only sets proc_title and so on. 

The console command already knows what it's executed from the console. If a console command started its own process, the result would be two processes per command.